README.md

```diff
@@ -90,3 +90,37 @@ await mongoose.connect(url, defaultMongoOpts)
 const person = await PersonRepo.byID(id));
 const people = await PersonRepo.all({});
 ```
+
+## How to run a migration
+
+```shell
+yarn migrations init
+```
+
+The above command will initialize a `migrate-mongo-config.js` in the root directory and create a migration folder in your `src` folder.
+
+Now run
+
+```shell
+yarn migrations create <name>
+```
+
+The command helps to create a migration script in the `src -> migration` directory as below:
```

Comment:
![image](https://user-images.githubusercontent.com/45825009/124656122-b2238680-de98-11eb-91ed-7c14be87314e.png)

The options in `.action((description, options) => {}` is responsible for changing the src/migrations directory. But how is it been passed with the command? as the command seems to only take `[description]`.

Reply:
I've gone through the code and I myself don't know why. Could use a rewrite at some point(the source library `migrate-mongo` takes care of this). For now though we're fine. I'll suggest you remove the `--file` option  since it's useless